### PR TITLE
Remove bower

### LINKED ISSUE #81: Remove Bower
We switched to npm

### PR DESCRIPTION
**Pull request by peterbraden**
Fixes #81

### CODE REVIEW

**Comment by peterbraden** [comment]
I think that most of the devDependencies should actually be dependencies. If you can't run the main task (gulp build) without these existing, then they are actually dependencies. At some point we're going to want NODE_ENV=production and at this point the devDependencies won't be installed.

**Comment by meodai** [comment]
as I understand it, is dependencies are things that someone who would like to see the built thing needs, and devDependencies are the things that you need to build the project.

**Comment by peterbraden** [comment]
we don't check in built artifacts, so without gulp build this has no functionality.

dependencies should be the _default_ and devDependencies should be use to optimise the module by removing unit testing etc that a consumer of the module won't need.

This is kind of a weird case as a boilerplate isn't really a module, but because the core _purpose_ of this module is to _build_ a static site, modules that are required to do that are core dependencies.

File: gulp/tasks/styles.js [hunk 1]
```diff
@@ -14,7 +13,7 @@ module.exports = function (gulp, $, config) {
         extensions: ['.scss']
       }))
       .pipe($.sourcemaps.init())
-      .pipe($.sass({includePaths: [bowerFiles]}))
+      .pipe($.sass({includePaths: []}))
```
Comment (lejoe) [inline gulp/tasks/styles.js]: So sass goes search the files magically in the npms?

File: package.json [hunk 2]
```diff
@@ -4,12 +4,11 @@
   "version": "0.0.0",
   "main": "dist/index.html",
   "scripts": {
-    "postinstall": "node_modules/bower/bin/bower install",
-    "postupdate": "node_modules/bower/bin/bower install"
   },
-  "dependencies": {},
+  "dependencies": {
+    "sensible": "^0.5.4"
```
Comment (lejoe) [inline package.json]: This should be fixed to "0.5.4"
c.f. #93

**Comment by lejoe** [comment]
@peterbraden ping. want to merge this. Do you have a sec to answer my feedback?

**Comment by peterbraden** [comment]
sorry, wifi here is atrocious, lemme look.

File: package.json (hunk 2 above)
Comment (peterbraden) [inline package.json]: :+1:

File: gulp/tasks/styles.js (hunk 1 above)
Comment (peterbraden) [inline gulp/tasks/styles.js]: that's the idea, yes. You explicitly require the styles you want however, in your index style file, as we do now.

**Comment by peterbraden** [comment]
Done, Not sure what's failing in the snap-ci, but I assume it's because we're calling bower when it isn't installed. I can dig in, but imho, this is still good to merge, and we can fix CI after.

**Comment by lucalanca** [comment]
@peterbraden: how can we install modules that are only on bower through npm?

**Comment by peterbraden** [comment]
You can specify github addresses in package.json. This is also the way to handle internal dependency's (private code).

**Pull request merged**